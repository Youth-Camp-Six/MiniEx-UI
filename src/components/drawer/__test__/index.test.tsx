/*eslint-disable*/
/**
 * @jest-environment jsdom
 */

import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { Drawer } from '..';

describe('Drawer', () => {
  // 快照测试
  it('test Drawer ', () => {
    const wrapper = render(<Drawer onClose={() => { }}>内容</Drawer>);
    expect(wrapper.container.firstChild).toMatchSnapshot();
  });
});
