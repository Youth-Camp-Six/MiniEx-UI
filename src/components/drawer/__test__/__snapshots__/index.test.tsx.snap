// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drawer test Drawer  1`] = `null`;
